conftest.py

```diff
@@ -288,41 +288,40 @@ def check_circuit_precision():
     return check_circuit_precision_impl
 
 
-def check_array_equality_impl(actual: Any, expected: Any, verbose: bool = True):
-    """Assert that `actual` is equal to `expected`."""
-
-    assert numpy.array_equal(actual, expected), (
-        ""
-        if not verbose
-        else f"""
+@pytest.fixture
+def check_array_equal():
```

Comment:
just a rename to be coherent with `numpy.array_equal` method (we should probably replace all `numpy.array_equal` with `check_array_equal` by the way)